notebooks/remote_functions/remote_function_usecases.ipynb

```diff
@@ -0,0 +1,1134 @@
+{
```

Comment:
<p>Likewise, some explanation that this is only for built-in and there is an example of third-party packages later on.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/808/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Added.</p>